src/components/tree/tree.js

```diff
@@ -35,21 +35,25 @@ class Tree extends React.Component {
       tree: null,
       treeToo: null
     };
-    /* bind callbacks */
-    this.clearSelectedNode = callbacks.clearSelectedNode.bind(this);
-    // this.handleIconClickHOF = callbacks.handleIconClickHOF.bind(this);
-    this.redrawTree = () => {
-      this.props.dispatch(updateVisibleTipsAndBranchThicknesses({
-        root: [0, 0]
-      }));
-    };
-    /* pressing the escape key should dismiss an info modal (if one exists) */
-    this.handlekeydownEvent = (event) => {
-      if (event.key==="Escape" && this.props.selectedNode) {
-        this.clearSelectedNode(this.props.selectedNode);
-      }
-    };
   }
+
+  clearSelectedNode = () => {
+    callbacks.clearSelectedNode.bind(this);
```

Comment:
[reminder to me] - check this change, I remember binding callbacks in the constructor was "the right way" when I was writing this code.

Reply:
I see:

https://github.com/nextstrain/auspice/blob/3cd0c1e60cd728e7c9bfc05f5d80b2607ba5af52/src/components/map/map.js#L84-L85

I've replaced 1eedee40 with a1aec383.